cloud/passportauth/src/auth/DefaultStrategy.ts

```diff
@@ -1,7 +1,8 @@
-import { BunyanLogger, Logger } from '@raincatcher/logger';
```

Comment:
RAINCATCH-1057 Change usage of the logger module in @raincatcher/passport-auth

I would say its covered